adapters/taboola/taboola.go

```diff
@@ -236,12 +228,30 @@ func createTaboolaRequests(request *openrtb2.BidRequest) (taboolaRequests []*ope
 		modifiedRequest.BAdv = taboolaExt.BAdv
 	}
 
+	if taboolaExt.PageType != "" {
+		modifiedRequest.Ext = makeRequestExt(taboolaExt.PageType, errs)
+	}
+
 	taboolaRequests = append(taboolaRequests, overrideBidRequestImp(&modifiedRequest, nativeImp))
 	taboolaRequests = append(taboolaRequests, overrideBidRequestImp(&modifiedRequest, bannerImp))
 
 	return taboolaRequests, errs
 }
 
+func makeRequestExt(pageType string, errs []error) json.RawMessage {
+	requestExt := &RequestExt{
+		PageType: pageType,
+	}
+
+	requestExtJson, err := json.Marshal(requestExt)
+	if err != nil {
+		errs = append(errs, err)
+		return make([]byte, 0)
```

Comment:
Would you like to continue making the taboola request if json marshalling fails here for any reason? If the intention is to not make the taboola request if makeRequestExt errors out then I would suggest to have this function return an error and handle that returned error in the caller(Line 232).

Reply:
Thanks for your comment, pushed a fix